packages/terra-data-grid/tests/wdio/data-grid-spec.js

```diff
@@ -280,6 +264,26 @@ Terra.describeViewports('DataGrid', ['medium', 'large'], () => {
       Terra.validates.element('data-grid-focusable-select-retains-focus', { columnResizeSelector });
       expect(browser.$('#specialties').isFocused()).toBe(true);
     });
+
+    it('focuses on a button if it is the only component in a cell', () => {
```

Comment:
Should we have a test with more than one focusable component in a cell?

Reply:
I think that should be covered here:

https://github.com/cerner/terra-framework/blob/b4b9f88804cb4484fc16269a650c55c56f299bdb/packages/terra-data-grid/tests/wdio/data-grid-spec.js#L234-L239